### PR TITLE
chore: prepare for release

### LINKED ISSUE #1614: MACI audit v2.0.0
Support MACI audit for v2.0.0 version release

### PR DESCRIPTION
**Pull request by 0xmad**
# Description

- [x] Add new download link for test artifacts
- [x] Update website docs

## Additional Notes

N/A

## Related issue(s)

Closes #1614 

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: website/.gitignore [hunk 1]
```diff
@@ -22,3 +22,5 @@ yarn-error.log*
 
 versioned_docs/version-v2.0_alpha/developers-references/typescript-code/typedoc/
```
Comment (ctrlc03) [inline website/.gitignore:23]: should we remove those 2?

File: website/versions.json [hunk 2]
```diff
@@ -1 +1 @@
-["v2.0_alpha", "v1.2", "v0.x"]
+["v2.x", "v1.2", "v0.x"]
```
Comment (ctrlc03) [inline website/versions.json:1]: not sure if it's just me but I cannot see the left navigation bar for the docs on v2.x

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
Thanks, left some comments, also let's bump to 2.0 on the various package.json from 2_alpha?

File: website/versions.json (hunk 2 above)
Comment (ctrlc03) [reply website/versions.json:1]: nice, it shows now for me too, not sure what happened there haha

**Review by kittybest — APPROVED** [review]
Thanks!

**Comment by 0xmad** [comment]
Let's wait for audit report updates

**Review by ctrlc03 — APPROVED** [review]

**Pull request merged**